sisl/io/wannier90/seedname.py

```diff
@@ -53,14 +54,14 @@ class winSileWannier90(SileWannier90):
     def _setup(self, *args, **kwargs):
         """ Setup `winSileWannier90` after initialization """
         self._comment = ['!', '#']
-        self._seed = self.file.replace('.win', '')
+        self._seed = str(self.file).replace('.win', '')
```

Comment:
perhaps `_seed` should also be a `Path`, then `_seed.with_suffix('.win')` would work

Reply:
I don't know if it every happens, but can't the seed contain a dot? In that case we'd be overwriting the last part.

Reply:
Doing 
```python
a = pl.Path('soaethuoase.')
a.with_suffix('.win')
PosixPath('soaethuoase..win')
```
yields the correct thing. :)